platform/ui/src/components/SegmentationGroupTable/SegmentationConfig.tsx

```diff
@@ -13,6 +13,15 @@ const ActiveSegmentationConfig = ({
   setFillAlpha,
   usePercentage,
 }) => {
+  const [
```

Comment:
The InputRange now reacts to changes to the value property (as it should). But here I noticed that the `usePercentage` property is NOT kept in sync with the various (config) values. What I mean is take for example the fill alpha. Initially it is a decimal and is converted via * 100. However, once the InputRange is used to alter the value, it is set as a value between 0-100 even though initially it was 0-1.0, but `usePercentage' is still true. Is there a better way to keep the association than what I have here?